external/date.js

```diff
@@ -228,7 +228,7 @@ $.extend( _Date.prototype, {
 	equal: function( other ) {
```

Comment:
ps: Removed all unused methods but not "equal()" as its used within the date object itself.